Libraries/react-native/react-native.js

```diff
@@ -172,6 +173,11 @@ const ReactNative = {
       return require('update');
     },
   },
+
+  // better error messages when accessing React APIs on ReactNative
+  get render() { throwOnWrongReactAPI('render'); },
```

Comment:
@javache thanks. will make the method contents __DEV__ only. I don't think I can do that for the whole block.